sloop-helm-charts/README.md

```diff
@@ -0,0 +1,9 @@
+# Use sloop-helm-charts to deploy your service
+1. Build image and tag, and replace the current ones in values.yaml, we are working an image for people to use, it will be release in a few days.
+2. Create a namespace in your cluster for sloop to run, for example: `kubectl create namespace sloop `
```

Comment:
nit: * only if you dont already have a namespace

Reply:
our current helm chart use namespace is default, if people dont change it, they will use default namespace